lib/menu.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'dish'
+require_relative 'formatter'
+
+class Menu
+  include Formatter
+
+  def initialize(dish_klass = Dish, dishes_str = nil)
+    @dish_klass = dish_klass
+    @dishes = Array.new
+    filename = "menu.txt"
+    dishes_str = read_file(filename) if dishes_str.nil?
+    parse_dishes(dishes_str)
+  end
+
+  def formatted_dishes
+    list = ""
+    @dishes.each_with_index do |dish, index|
+
+      num, name, price = (index).to_s, dish.name, pound_str(dish.price)
```

Comment:
Style/ParallelAssignment: Do not use parallel assignment. (https://github.com/bbatsov/ruby-style-guide#parallel-assignment)